examples/docker-compose.socket-proxy.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  service:
+    image: containrrr/shepherd
+    environment:
+      DOCKER_HOST: tcp://tasks.socket-proxy:2375
+    networks:
+      - socket_proxy
+    deploy:
+      replicas: 1
+      
+  socket-proxy:
+    image: tecnativa/docker-socket-proxy
+    networks:
+      - socket_proxy
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    deploy:
+      replicas: 1
+      placement:
+        constraints:
+          - node.role==manager
+    environment:
+      LOG_LEVEL: info
+      POST: 1         # Enable methods that allow changes
+      ## Variables match the URL prefix (i.e. AUTH: 1 allow access to /auth/* parts of the API, etc.).
+      # AUTH: 1       # Enable this if you're using registry authentication
+      SERVICES: 1
+      DISTRIBUTION: 1
+      TASKS: 1
+      NODES: 1
```

Comment:
Does shepherd really need also those permissions?

Reply:
Yep, I tested each one. If you want to verify, just watch the socket-proxy service log. If an API is blocked, you'll see NOSRV. Then, you just enable that API.